src/pages/VeSDL/VeTokenCalculator.tsx

```diff
@@ -122,15 +122,18 @@ export default function VeTokenCalculator({
             select
             SelectProps={{ IconComponent: ArrowDownIcon }}
           >
-            <MenuItem value="">
-              <em>None</em>
-            </MenuItem>
             {basicPools &&
-              Object.keys(basicPools)?.map((poolName) => (
-                <MenuItem key={poolName} value={poolName}>
-                  {poolName}
-                </MenuItem>
-              ))}
+              Object.keys(basicPools)?.map((poolName) => {
+                const pool = basicPools[poolName]
+                const outdated =
+                  pool?.isGuarded || pool?.isMigrated || pool?.isPaused
+                if (outdated) return null
+                return (
+                  <MenuItem key={poolName} value={poolName}>
```

Comment:
poolNames unique?

Reply:
We can use pool address instead of pool name.

Reply:
Udated!